src/main/msp/msp_protocol_v2_common.h

```diff
@@ -23,3 +23,4 @@
 
 // Sensors
 #define MSP2_SENSOR_GPS                 0x1F03
+#define MSP2_SENSOR_RANGEFINDER         0x1F01
```

Comment:
Better make the name more complicated, or it will become de-facto standard for rangefinders.
```suggestion
// TODO: implement new, extensible rangefinder protocol
#define MSP2_SENSOR_RANGEFINDER_LIDARMT         0x1F01
```